classes/job/repositories/FailedJob.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file classes/job/repositories/Job.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2000-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Job
+ *
+ * @brief Job Repository
+ */
+
+namespace PKP\job\repositories;
+
+use Carbon\Carbon;
+use Illuminate\Contracts\Support\Arrayable;
+use Illuminate\Pagination\LengthAwarePaginator;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Collection;
+use PKP\facades\Repo;
+use PKP\job\repositories\BaseRepository;
+use PKP\job\models\FailedJob as PKPFailedJobModel;
+use PKP\job\resources\CLIFailedJobResource;
+use PKP\job\resources\HttpFailedJobResource;
+
+class FailedJob extends BaseRepository
+{
+    public function __construct(PKPFailedJobModel $model)
+    {
+        $this->model = $model;
+    }
+
+    public function showJobs(): LengthAwarePaginator
```

Comment:
This should be moved out of the model (`classes/job`) and into the view.

Reply:
@asmecher bit confused about the requested change. `showJobs` method resides in the repository not model . Also not sure what you mean by view .